tests/test_project/tests/test_admin.py

```diff
@@ -52,17 +54,21 @@ def test_alwayshaschangedmixin(self):
         self.assertEqual(Project.objects.count(), 1)
         self.assertEqual(Operator.objects.count(), 1)
         project = Project.objects.first()
+        operator = Operator.objects.first()
         self.assertEqual(project.name, params['name'])
+        self.assertEqual(operator.first_name, 'test')
+        self.assertEqual(operator.last_name, 'test')
 
         change_params = {
             'name': 'test',
+            'key': self.TEST_KEY,
             'operator_set-TOTAL_FORMS': 1,
             'operator_set-INITIAL_FORMS': 1,
             'operator_set-MIN_NUM_FORMS': 0,
             'operator_set-MAX_NUM_FORMS': 1000,
             'operator_set-0-first_name': 'test2',
             'operator_set-0-last_name': 'test2',
-            'operator_set-0-id': project.pk,
+            'operator_set-0-id': operator.id,
```

Comment:
There was a mistake here. It should be operator's id instead of projects. Previously, `project.pk` was just a number, which coincidentally was also equal to 1, but now `pk` is a uuid, therefore the test failed.

Reply:
good call